README.md

```diff
@@ -499,85 +367,219 @@ i18n.loadLocale(locale, params)
 i18n.runWithLocale(locale, func)
 // it means that every default locale used inside a called function will be set to a passed locale
 // keep in mind that locale must be loaded first (if it is not bundled)
+```
 
-// additional informations about locale (locale is by default set to the current one)
+## Integration with React
 
-getCurrencySymbol (code) // or locale with country
-returns currency symbol if known
+If you want to use this package with React, you need to create two functions `createTranslator` and `createComponent`. Example code is below.
```

Comment:
The entire React integration example is far too complex. Let's make one using hooks instead.